lib/assayer.js

```diff
@@ -1,54 +1,51 @@
-'use strict'
+'use strict';
 
-const geolocate = require('./assays/geolocate')
```

Comment:
Lot of linting in this file, but [here is the change](https://github.com/PRX/analytics-ingest-lambda/pull/95/commits/814ebf26efaf26439f12b078a7527fcc4a1e8b76#diff-5e85ec8ac0dbb5774926fe26051bd99bc7006ec3832bba6b5105760f73724e2cL3).  Just removing the geolocate.